### DISCUSSION TITLE
public servers are beeing useless since the unofficial servers are added.

### DISCUSSION THREAD

**Discussion by lucaschambers** [BearWare/TeamTalk5#1849] [Ideas]
Hello there! I wanted to bring to your attention that the public servers in the team talk application are becoming less reliable since the addition of unofficial servers. As a solution, I propose that you remove the entire public server feature and give users the ability to select their preferred language or country in order to find servers within their desired communities. This way, users can find as many servers as they want, similar to other conferencing applications.
Also, the developer doesn't need to bother themselves with adding or removing servers from the public servers all the time. This is because anyone can add their servers to the unofficial section, and people can find their community easier this way.
Thank you for considering this suggestion, and please let me know if you have any questions or comments.

**Comment by bear101** [comment]
I'm also in favor of removing public servers. It takes a lot of time to argue with people whether their server should be on the public server list or not. So far I've tried to limit the number of public servers by making restrictions on country and language.

In TeamTalk v5.13 it is possible to filter servers by name and/or number of active users. That way it's possible to limit the length of the public server list.

**Reply by lucaschambers** [reply]
Hi there, I mean it's better to remove the entire public server feature and give users the ability to host their servers from their country as many as they want. For example, let's say I'm from America, and I see only one server from the US, but I'm actually looking for more servers in the US with different admins or different communities. I look at the unofficial servers and see more servers in the United States. So if you remove the entire public server feature, you'll be giving a chance to other communities to show themselves, and a specific server won't be exclusively for just one country. People will have a chance to get connected with others in the same country. 
In the unofficial server's section, people can host their servers with no country limitations, and they can add their servers automatically, without arguments about whether the server should stay on the public server's list or not. I think public servers are now even confusing for some people, and I would suggest you remove the entire public server feature, and keep the official servers and unofficial servers that are being hosted by other people. 
You can also add an ability so that if a server has more popularity, it can get on top of the list, making it easier for people to connect with each other. Thank you for considering this suggestion."

**Comment by CoBC** [comment]
I'm also agree with removal of public servers. Official and unofficial are probably sufficient. More, it seams that some users are a bit confused with this three options, especially when they have to check and uncheck checkboxes to choose which ones they want to include in the server list.

**Reply by lucaschambers** [reply]
I agree100%

**Reply by kirill-jjj** [reply]
I agree, only I would remove the public ones and rename unofficial servers to public

**Reply by kirill-jjj** [reply]
and I would also suggest that users choose the country themselves, otherwise, for example, my vds is located in Spain, and I create a server for Ukrainians
